sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobRange.java

```diff
@@ -17,6 +17,15 @@
     private final long offset;
     private final Long count;
 
+    /**
+     * Specifies the download operation to start from the offset position (zero-based) and download the
+     * rest of the entire blob to the end. An empty BlobRange effectively specifies the whole blob.
+     */
+    public BlobRange() {
```

Comment:
Was this added to simplify some code? I'm not completely against it but given the immutability of this class it is easily replicated by `new BlobRange(0)`

Reply:
You're right. Just removed it